src/fides/api/schemas/pre_approval_webhooks.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+
+from fideslang.validation import FidesKey
+
+from fides.api.schemas.base_class import FidesSchema
+from fides.api.schemas.connection_configuration.connection_config import (
+    ConnectionConfigurationResponse,
+)
+
+
+class WebhookBase(FidesSchema):
+    """Base schema for Webhooks"""
+
+    key: Optional[FidesKey]
+    name: Optional[str]
+
+
+class PreApprovalWebhookCreate(WebhookBase):
```

Comment:
Should we make the name required on create? I see this is patterned off of the Policy webhooks but we get 500 errors without it.

Reply:
hm, yes, that makes sense, as the table requires it and we don't generate a "name" if empty 👍